src/Components/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,49 @@
+
+.checkbox-wrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.checkbox-label-wrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.task-item-checkbox:hover {
+  cursor: pointer;
+}
+
+input[type="checkbox"] {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 20px;
+  height: 20px;
+  opacity: 0;
+  // display: none;
+}
+
+.checkmark {
+  width: 20px;
+  height: 20px;
+  border-radius: 6px;
+  border: 2px solid var(--secondary-color-2);
+  z-index: 2;
+}
+
+.checkmark:hover {
+  border: 2px solid var(--secondary-color-1);
+  cursor: pointer;
+}
+
+input[type="checkbox"]:checked + .checkmark {
+  background: var(--primary-color-2);
```

Comment:
I would recomend to use naming convention similar to that from the screen at the end with eventual usage of saturation indication (i.e primary-100, primary-500, primary-900 - where 100 means least saturated, 500: normal variant and 900 :  most saturated). Its tailwind convention widely used, something that will be usefull for the future for you. But it's up to you 
![obraz](https://github.com/macopul/Todo-App-React/assets/21220135/7fff26ee-6505-4eac-8260-902c9223a58b)

Reply:
You mean like base-100, base-200, base-300 from example? 
In this case those 3 are completely different colors, or it is the same color, but somewhere specified with tone of this color ?

In our example can I just rename colors used in project (even if their tones don't match) or have I to generate somehow different tones of colors for this purpose ?

If there are so different colors for example -in --secondary colors, then should I extract them to --tertiary e.t.c.  ?